code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -265,3 +265,8 @@
 /obj/item/reagent_containers/pill/breast_enlargement
 	name = "breast enlargement pill"
 	list_reagents = list(/datum/reagent/fermi/breast_enlarger = 10)
+
+/obj/item/reagent_containers/pill/genital_conflictions
+	name = "genital confliction pill"
+	desc = "One of the strangest concoctions ever thought up by Euan Morrison, a pill containing both penis enlargement and breast enlargement reagents at a high dosage. This causes an... interesting reaction when administered."
+	list_reagents = list(/datum/reagent/fermi/penis_enlarger = 20, /datum/reagent/fermi/breast_enlarger = 20)
```

Comment:
You can literally just mix them both and make the pills yourself. Why would we want to add more hornycode stuff when we're aiming to tone that down?

Reply:
> You can literally just mix them both and make the pills yourself. Why would we want to add more hornycode stuff when we're aiming to tone that down?

i can and do make the pills myself in-game, i'm just making code for it because it would be interesting lore.
a local chemist's new drug interests the company who's reagents they're mixing together, and pay for it to be added to the vendor's inventory.

Reply:
h a m p t e r